docs/source/quickstart-aws.rst

```diff
@@ -43,17 +43,10 @@ In order to complete this quickstart, you will need to be familiar with the AWS
 Get ElasticBLAST
 ----------------
 
-Copy and paste the commands below at the CloudShell prompt to install ElasticBLAST.
-
-.. code-block:: shell
-
-    sudo yum install -y python3-wheel
-    sudo pip3 install elastic-blast
-
-
-In some cases (e.g., not on the cloud), it may be preferable to run these commands without using "sudo" (which runs these commands with root permissions).  
-
-The instructions in this quickstart assume that you are working from the directory where you installed ElasticBLAST.
+Please follow the instructions on :ref:`tutorial_pypi` or
```

Comment:
I think that we should have the lines of code here on this page, rather than require users to follow a link.

Reply:
I agree.  For the quickstart, people should not have to navigate away to look things up.

Reply:
Thanks for the feedback, this is done in b87e1a4.